### PR TITLE
Introduce a query for obtaining solver document ids based on package

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

With this query, we are able to obtain all the solver document ids that correspond to solving the given package.

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -799,6 +799,46 @@ def _construct_solved_python_packages_query(
         kwargs.pop("self", None)  # static method
         return self.__class__._construct_python_packages_query(**kwargs)
 
+    def get_solver_document_id_for_package_version(
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:802]: Shouldn't be just another filter for: https://github.com/thoth-station/storages/blob/b29d1606d5dce208e18012e456900810293284ed/thoth/storages/graph/postgres.py#L4323

Comment (fridex) [reply thoth/storages/graph/postgres.py:802]: +1, good catch. Will move them there.

**Comment by fridex** [comment]
/close

I think we do not need to ship this functionality.

**Pull request closed without merging**